lib/matplotlib/backends/backend_webagg_core.py

```diff
@@ -215,6 +223,11 @@ def get_diff_image(self):
             # Swap the renderer frames
             self._renderer, self._last_renderer = (
                 self._last_renderer, renderer)
+
+            self.figure._cachedRenderer = self._renderer
+            for ax in self.figure.axes:
```

Comment:
we don't cache the renderer on the axes anymore, so these two lines should go away